doc/source/examples/quickstart.ipynb

```diff
@@ -1,28 +1,29 @@
 {
```

Comment:
<p>I wonder why you're getting this simtk deprecation warning? Are you somehow importing parmed as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Fixed.</p>